x-pack/plugins/secops/public/containers/who_am_i/index.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { get } from 'lodash';
```

Comment:
We can discuss this further, but my preference would be to only import from `lodash/fp`.

In this particular case, (but others as well), `lodash/fp` has an even better API: `getOr`

Using `getOr` changes this:
```
children({
  appName: get(data, 'source.whoAmI.appName') || 'Who am I ?',
})
```

to this:

```
children({
  appName: getOr('Who am I ?', 'source.whoAmI.appName', data) // 98% sure the default value is the first param, but see the docs
})
```

Reply:
👍